docker-compose.yml

```
@@ -85,8 +85,10 @@ services:
     image: obolnetwork/charon:${CHARON_VERSION:-v1.3.0}
     environment:
       - CHARON_BEACON_NODE_ENDPOINTS=${CHARON_BEACON_NODE_ENDPOINTS:-http://lighthouse:5052}
+      - CHARON_BEACON_NODE_HEADERS=${CHARON_BEACON_NODE_HEADERS:-Dummy=Header}
```

Can we have empty one here instead of always sending this header from now on?

```suggestion
      - CHARON_BEACON_NODE_HEADERS=${CHARON_BEACON_NODE_HEADERS:-}
```

---

IIRC that caused an error for me but maybe that's been fixed